### DISCUSSION TITLE
how to report current qps/status in interval?

### DISCUSSION THREAD

**Discussion by woodgear** [fortio/fortio#982] [Q&A]
if i set a long run duration, there no output in stdout. maybe report the current status in a interval?

**Comment by ldemailly** [comment]
You can use the `-access-log-file` (and format) to log every request and their duration and ingest that in a timeseries DB (like influxdb)

Or use fortiotel variant which sends trace to open telemetry

Or use fortio as a library or script it such as you cut your long running run into smaller segments

Other realtime updates would be #278 / #438

**Comment by ldemailly** [comment]
and as this was a question I moved it from Issue to Discussion

**Comment by woodgear** [comment]
that mean fortio have to depend on those timeseries DB.. i thought it should better to just report it in stdout..

**Reply by ldemailly** [reply]
log file is... just a file being written, there is no direct dependency on any specific DB but just a file format that matches one. the access logger is also pluggable. Likewise opentelemetry traces can talk to a number of backends, it's an open standard.

One reason none of this is on by default is that it impacts the throughput. Fortio can do 100k qps on my laptop but not with logging every call, nor if it had to synchronize to emit stats before the end

Yet it's something or a mode to consider
